standards_lab/processor/cove.py

```diff
@@ -11,17 +13,31 @@
 import tempfile
 
 import django_rq
+import jsonref
 from rq.job import Job
 from rq.exceptions import NoSuchJobError
 
+import api.views
 from .extra_validator_funcs import patch_validator
 
 
 patch_validator(validator)
 
 
-def start(project):
+MIME_TYPE_TO_FILE_TYPE = {
+    "application/csv": "csv",
+    "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet": "xlsx",
+    "application/vnd.oasis.opendocument.spreadsheet": "ods",
+}
+
+
+def lib_cove_wrapper(
+    project,
+    data_file,
+    cache=False,
+):
     schema_name = project["rootSchema"]
+    root_list_path = "grants"
```

Comment:
@michaelwood This value is currently hardcoded to test 360 data (as this PR is still WIP). We need to get this value somehow, possibly just via a text box in the UI. One to discuss tomorrow morning maybe.